x-pack/legacy/plugins/apm/server/lib/transactions/distribution/get_buckets/fetcher.ts

```diff
@@ -58,7 +58,7 @@ export function bucketFetcher(
             min_doc_count: 0,
             extended_bounds: {
               min: 0,
-              max: bucketSize * bucketTargetCount
+              max: distributionMax
```

Comment:
The previous `max` value was calculated from the rounded `bucketSize` so it was not accurate and would therefore miss buckets. Using the exact `distributionMax` from the stats request fixes this.